scripts/seed.js

```diff
@@ -533,11 +570,60 @@ async function createPermissions() {
         object: 'agora',
         scope: 'global',
         description: 'Update the is_on_memberslist status for applications for Agora. Should be assigned to Network Director.'
+    },
+    {
+        action: 'manage_network',
+        object: 'communication_exception',
+        scope: 'global',
+        description: 'Give exceptions to the `communication` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'board_election',
+        scope: 'global',
+        description: 'Set the fulfilment of the `board election` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'members_list',
+        scope: 'global',
+        description: 'Set the fulfilment of the `members list` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'events',
+        scope: 'global',
+        description: 'Set the fulfilment of the `events` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'agora_attendance',
+        scope: 'global',
+        description: 'Set the fulfilment of the `agora attendance` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'development_plan',
+        scope: 'global',
+        description: 'Set the fulfilment of the `development plan` Antenna Criterion'
+    },
+    {
+        action: 'manage_network',
+        object: 'fulfilment_report',
+        scope: 'global',
+        description: 'Set the fulfilment of the `fulfilment report` Antenna Criterion'
     }], { individualHooks: true, validate: true });
 
     permissions.networkDirector = [...networkDirectorPermissions, ...permissions.netCom, permissions.seeMemberslistsAgora, permissions.setMemberslistsFeePaidAgora];
```

Comment:
```suggestion
    permissions.networkDirector = [...networkDirectorPermissions, ...permissions.netCom, permissions.setMemberslistsFeePaidAgora];
```
NetCom has that role already